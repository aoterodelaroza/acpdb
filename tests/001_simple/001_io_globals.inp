## check: 001_io_globals.out -a1e-10
## labels: regression quick nodata

# this is a comment
echo \
 bleh \


