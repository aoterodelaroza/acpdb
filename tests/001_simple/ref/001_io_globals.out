* NCPU set to 4

* MEM set to 2GB

